### DISCUSSION TITLE
Can you give me your contact information?

### DISCUSSION THREAD

**Discussion by qintian9188** [openbullet/openbullet#909] [General]
Can you give me your contact information?

**Comment by openbullet** [comment]
You can find me on the official forum, or you can send me an email. The address is in the readme of the repo

**Reply by qintian9188** [reply]
Can I contact you here?

**Reply by openbullet** [reply]
No

**Comment by qintian9188** [comment]
why？


openbullet ***@***.***> 于2024年7月31日周三 00:18写道：
